Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,387 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+
+## Detail Design: Resistance System ##
+
+
+### Function of the Subsystem:
+
+The function of the resistance system is to control and vary the resistance felt by the user of the Mario Kart Bike, hereafter referred to as the bike.  
+
+- The controller monitors the linear actuator feedback voltage to determine its current position with 10-bit precision, which corresponds to the distance of the magnet array from the flywheel. 
+
+- An external digital input (via the Bluetooth module) is received from the ride replay subsystem to determine the calculated set point which the actuator should extend to. This recieved value corresponds to a resistance state, of which there are 511; the limiting factors being the 10-bit precision of the controller and the usable portion of the actuator's range.
+
+- The controller supplies either a positive or negative voltage to the linear actuator to control the direction of the actuator's motion.
+
+- The actuator is to vary the array of magnets perpendicular to the simulator’s rear conductive flywheel (not the current flywheel on the bike, a new flywheel with dimensions that fit our design more effectively) to generate eddy currents using the primary magnetic induction formed at the conductor [1], which result in the generation of braking torque opposed onto the flywheel. 
+
+- The braking torque will transfer from the flywheel to the rear bike tire, and then from the rear bike tire to the rider’s pedals, which will act to resist the pedaling of the rider. 
+
+
+
+### Constraints:
+
+#### Magnet Distance Range: 
+
+- A distance range of 2-25mm from magnet to flywheel was selected, as these distances matched up well with the torque values that we want the resistance system to apply. 
+
+ 
+
+#### Minimum Tire Speed: 
+
+- The minimum speed of the bike tire shall be considered 116.35 rpm. This value is based on the American Council on Exercise's recommendation for the minimum speed required for moderate exercise on a bike [2]. This correlates to 2501.52 rpm in the flywheel (conversion is explained in later sections). At speeds less than 116.35 rpm, the torque placed on the rider will not be specified to reach any minimum value, and the resulution of the resistance changes within this operating region is not specified.
+
+ 
+
+#### Maximum Tire Speed: 
+
+- The maximum speed of the bike tire shall be considered 288 rpm, a value that comes from the average pedaling rpm (118 rpm) of an elite level cyclist [3] (Wheel to pedal gear ratio = 2.444, so wheel speed = 2.444*pedal speed). This correlates to 6192 rpm in the flywheel due to the the rotational velocity ratio of 43:2 between the flywheel and the wheel. If the user passes this speed, no additional torque will be applied to avoid injury to rider. 
+
+ 
+
+#### Maximum Torque at Minimum Speed: 
```

Comment:
This makes sense. My question is, is that sufficient? By that I mean, does that meet your requirements?